src/mapml/layers/TemplatedFeaturesLayer.js

```diff
@@ -76,14 +80,40 @@ export var TemplatedFeaturesLayer =  L.Layer.extend({
         this._onMoveEnd();
     },
 
+    _removeCSS: function () {
+        let toDelete = this._container.querySelectorAll("link[rel=stylesheet],style");
+        for(let i = 0; i < toDelete.length;i++){
+            let parent = toDelete[i].parentNode;
+            parent.removeChild(toDelete[i]);
+        }
+    },
+
     _onMoveEnd: function() {
+      let history = this._map.options.mapEl._history;
+      let current = history[history.length - 1];
+      let previous = history[history.length - 2];
+      let step = this._template.step;
       let mapZoom = this._map.getZoom();
+      let steppedZoom = mapZoom;
+      if (((step !== "1") && ((mapZoom + 1) % step === 0) && current.zoom === previous.zoom - 1) ||
```

Comment:
When you first load the layer, previous can be undefined here, which happens when you press the "Reload" control button